tox.ini

```diff
@@ -14,7 +14,7 @@ description =
   Run the tests under {basepython} and
   devel: ansible devel branch
   ansible29: ansible 2.9
-  core: ansible-base 2.10
+  core: ansible-core 2.11+
   py: ansible-core 2.11
```

Comment:
?

Reply:
That is the description text and is correct, now "core" translates to ansible-core.

Reply:
Yes, but this seems to duplicate `core` which is weird.

Reply:
Is does not duplicate anything, the line you mentioning is using the "core" substring from tox env name in order to build a meaningful doc description for the environment. Not including it or updating it will mislead users when they run `tox -va`.

The reality is right now we are not testing the linter with 2.11, we only test it with `2.9`, `2.10` and `devel` while 2.10 is mislabeled as core. Once we merge this PR we will officially test latest core.

We could restore 2.10 testing if we want but there is no real incentive as the change of specific issue with it is close to zero if both others passed.